index.html

```diff
@@ -697,6 +698,36 @@ <h2>
   top: anchor(--my-anchor-name-b bottom);
 }
 </code></pre>
+    </section>
+    <section id="anchor-absolute" class="demo-item">
+      <h2>
+        <a href="#anchor-absolute" aria-hidden="true">🔗</a>
+        Absolutely positioned anchor
+      </h2>
+      <div style="position: relative; height: 150px;" class="demo-elements">
```

Comment:
I guess the linter complains about this extra semicolon 🤷 

```suggestion
      <div style="position: relative; height: 150px" class="demo-elements">
```